releng/org.eclipse.xtext.target/pom.xml

```diff
@@ -12,4 +12,36 @@
 
 	<packaging>eclipse-target-definition</packaging>
```

Comment:
@kthoms just in case, you can use "pom" as packaging and have builder-helper-maven plugin only attach the target file based only on the classifier, see, e.g., https://github.com/LorenzoBettini/swtbot-example/blob/master/swtbot.example.targetplatform/pom.xml

Reply:
Good idea. I think we could change this in that direction further.